backend-db/modules/db/couchdb/index.js

```diff
@@ -612,6 +624,92 @@ const layer = {
         }
         return false;
     },
+
+    //========================================================
+    // TEAMS
+    //========================================================
+    getTeam: async id => {
+        return await adapter.getOneEntityById('Team', '_design/all/_view/view', id);
+    },
```

Comment:
I know this is copy-paste, but it's technically double-wrapping promises. :) You can get rid of async and await, since all the content is just one line that returns a promise anyway:

    getTeam: id => adapter.getOneEntityById('Team', '_design/all/_view/view', id),